docs/howto/kubernetes/gardener/rolling-upgrades.md

```diff
@@ -0,0 +1,68 @@
+---
+description: How to manually upgrade machine images or Kubernetes versions for Gardener clusters
+---
+# Conducting rolling upgrades
+
+By default, Kubernetes clusters created with {{k8s_management_service}}
+in {{brand}} are upgraded automatically. Those upgrades take place
+during a specified maintenance window, and you may find out more about
+[what they involve and how they
+work](../../../../background/kubernetes/gardener/autoupgrades). Besides
+the automatic upgrades, you can manually apply any upgrades available
+for your cluster. In the following two examples, we first show how to
+manually upgrade the machine image the cluster nodes are based on and
+then how to upgrade the Kubernetes version the cluster uses.
+
+## Upgrading machine images
+
+In the left-hand vertical pane of the {{gui}}, select Containers →
+[{{k8s_management_service}}](https://{{gui_domain}}/containers/gardener).
+In the central pane, you will see all your {{k8s_management_service}}
+clusters, each on its own row. If a Kubernetes upgrade is available, you
+will notice, in the *Version* column, a triangular marker together with
+the new Kubernetes version. On the other hand, if there is a new machine
+image version for the cluster nodes, in the *Workers* column you will
+notice just a triangular marker --- no version number.
```

Comment:
Somehow, this word order would sit better with me:

```suggestion
clusters, each on its own row. If a Kubernetes upgrade is available, you
will notice a triangular marker together with
the new Kubernetes version, in the *Version* column.
On the other hand, if there is a new machine
image version for the cluster nodes, you will
notice just a triangular marker --- no version number --- in the *Workers* column.
```

Reply:
Indeed, I think your version is *less* Christos-English. I am taking note of your suggestion and propose an equally acceptable version (I think).